src/services/webPlatformUtils.service.ts

```diff
@@ -283,11 +288,24 @@ export class WebPlatformUtilsService implements PlatformUtilsService {
         return false;
     }
 
-    getDefaultSystemTheme() {
-        return Promise.resolve(null as 'light' | 'dark');
+    getDefaultSystemTheme(): Promise<ThemeType.Light | ThemeType.Dark> {
+        return Promise.resolve(this.prefersColorSchemeDark.matches ? ThemeType.Dark : ThemeType.Light);
+    }
+
+    async getEffectiveTheme(): Promise<ThemeType.Light | ThemeType.Dark> {
+        const theme = await this.storageService().get<ThemeType>(ConstantsService.themeKey);
```

Comment:
Do we need to default to `'system'` here?

Reply:
See response above